.github/workflows/ci-idris2-and-libs.yml

```diff
@@ -578,12 +593,86 @@ jobs:
         run: |
           make all
 
+  # PACK
+  #
+  # N.B. instead of bootstrapping pack, we use the dockerimage where it is
+  # already, and then use pack to rebuild pack with this job's idris2 sources
+
+  ub-pack-test-pack:
+    needs: ubuntu-self-host-previous-version
+    runs-on: ubuntu-latest
+    if: |
+      !contains(needs.initialise.outputs.commit_message, '[ci:')
+      || contains(needs.initialise.outputs.commit_message, '[ci: libs]')
+    env:
+      IDRIS2_CG: chez
+      PACK_DIR: /root/.pack
+    strategy:
+      fail-fast: false
+    # N.B.:
+    container: ghcr.io/stefan-hoeck/idris2-pack:latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          repository: 'stefan-hoeck/idris2-pack'
+      # by default, pack uses the main idris2 head, not the current job's one
+      - name: Toml setup
+        run: |
+          echo "[idris2]" > pack.toml
+          echo "url = \"https://github.com/${GITHUB_REPOSITORY}\"" >> pack.toml
+          echo "commit = \"latest:${GITHUB_REF_NAME}\"" >> pack.toml
+          echo "bootstrap  = true" >> pack.toml
+      - name: Build idris2-pack
```

Comment:
I think, after setting `pack.toml`, one needs to call `pack fetch` to make sure `pack` has fetched the correct commit of the branch mentioned in `latest:`

Reply:
The local testing I did with this seemed to correctly get the commit, but I'll @stefan-hoeck just to be sure; afaiu, `pack install pack` is clever enough to fetch the latest/specified Idris2-commit before rebuilding and reinstalling itself  : )

Reply:
My knowledge would be that in this case the latest commit at the moment of creation of the docker image would be used instead (without calling `pack fetch`, as I suggest), but right, let's wait for Stefan's answer

Reply:
Did some testing using the docker image rather than my local `pack` install; it seems `pack fetch` is indeed redundant:

Before, i.e. having just launched a fresh copy of the image:
```
# idris2 --version
Idris 2, version 0.6.0-72270962f
```

After creating a matching `pack.toml` and running `pack install pack`:
```
# idris2 --version
Idris 2, version 0.6.0-2a96c36fd
```

Which matches the latest commit on this branch  : )

Reply:
`git fetch` is needed when there were some commits into the `main` branch between the docker image creation and running the CI. Since there were no commits into `main` recently, you cannot reproduce this situation